snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml

```diff
@@ -54,6 +54,55 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.318.1.1.1.7.2.3
       name: upsAdvTestDiagnosticsResults
+  - MIB: PowerNet-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.1.11.1
+      name: upsBasicStateOutputState.OnLine
+    forced_type: flag_stream
+    options:
+      placement: 4
```

Comment:
What do you think about having the metric_suffix as an option ?

I was thinking to validate the `name` at some point against the `OID` using MIBs. 

And it’s seems semantically more correct since both placement and `metric_suffix` are options of `forced_type`: `flag_stream`

```
  - MIB: PowerNet-MIB
    symbol:
      OID: 1.3.6.1.4.1.318.1.1.1.11.1
      name: upsBasicStateOutputState
    forced_type: flag_stream
    options:
      placement: 4
      metric_suffix: OnLine
```

Good idea for reusing the forced_type.